src/models/generated/CarbonCaps.jl

```diff
@@ -5,91 +5,92 @@ This file is auto-generated. Do not edit.
 #! format: off
 
 """
-    mutable struct CarbonCaps <: Requirements
+    mutable struct CarbonCaps <: Requirement
```

Comment:
We should add a target_year Int type in this requirement. Ditto for CapacityReserveMargin, CarbonTax, MaximumCapacityRequirements, MinimumCapacityRequirements